_commands/basics/mkdir.md

```diff
@@ -0,0 +1,77 @@
+---
+---
+
+mkdir
+--
+
+`mkdir` is used to create directories (if they do not already exist) on a file system.
+
+The general format is
+
+~~~ bash
+$ mkdir [options] directories
+~~~
+
+A simple example is:
+~~~ bash
+$ mkdir dir1
+~~~
+This will create a folder named **dir1** under the current path.
+
+
+### Useful Options / Examples
+
+#### `mkdir -p,--parents`
+Create parent directories as necessary. (If the sepcified parent directory already exist, no error is reported.)
```

Comment:
This parenthetical part is also important (in fact it's the more common usage of `-p`). Could you add an example that shows this off, something like:

```bash
$ mkdir /tmp/test
$ mkdir /tmp/test
<error>
$ mkdir -p /tmp/test
```